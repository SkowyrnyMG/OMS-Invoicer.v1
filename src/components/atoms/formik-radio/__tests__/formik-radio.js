import React from 'react';

import {
  renderWithReduxRouter,
  FormikTestWrapper,
  screen,
} from 'utils/tests/test-helper';
import FormikRadio from '../formik-radio';

describe('FormikRadio', () => {
  test('should display in the document with correct styles', () => {
    renderWithReduxRouter(
      <FormikTestWrapper>
        <FormikRadio name='test_radio' value='test_value' />
      </FormikTestWrapper>,
    );

    const component = screen.getByRole('textbox', { name: 'test_value' });

    expect(component).toBeInTheDocument();
    expect(component).toMatchSnapshot();
  });
});
