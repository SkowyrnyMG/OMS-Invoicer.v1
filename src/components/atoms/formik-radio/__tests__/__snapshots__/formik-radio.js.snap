// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormikRadio should display in the document with correct styles 1`] = `
.c0 {
  position: absolute;
  opacity: 0;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
}

.c0:checked + label {
  font-weight: 700;
  opacity: 1;
}

<input
  class="c0"
  id="test_radio-test_value"
  name="test_radio"
  value="test_value"
/>
`;
